### PR TITLE
Added acf_clone example

### PR DESCRIPTION
**Pull request by edvwib**
Added acf_clone example. Tried to create the key myself on the original text input, that didn't work so I put the way I got the field key in a comment next to it.

### CODE REVIEW

**Comment by vinkla** [comment]
Try adding a custom key to the original field and then use that key. Let me know how it goes.

**Comment by edvwib** [comment]
That's what I

**Comment by edvwib** [comment]
That's what I tried to do, didn't work though...

**Comment by vinkla** [comment]
> That's what I tried to do, didn't work though...

Hmm, are you using version 6.0+?

File: README.md [hunk 1]
```diff
@@ -427,10 +427,22 @@ acf_accordion([
...
 ```
 
-**Clone** - The [clone field](https://www.advancedcustomfields.com/resources/clone) allows you to select and display existing fields.
+**Clone** - The [clone field](https://www.advancedcustomfields.com/resources/clone) allows you to select and display existing fields or groups.
 
 ```php
-// TODO: Add acf_clone function example.
+acf_text([
+	'label' => 'name',
+	'name' => 'Name',
```
Comment (vinkla) [inline README.md]: @edvwib did you try adding the `key` field here? This should work:

https://github.com/wordplate/acf/blob/ec5d5c2078ab2913edca1e43b249ffa3ab4eeb29/tests/FieldTest.php#L156-L181

Comment (edvwib) [reply README.md]: I got it to work, was using an outdated version of wordplate/acf (5.1) :man_facepalming:

File: README.md [hunk 2]
```diff
@@ -431,17 +431,19 @@ acf_accordion([
...
-	'placeholder' => 'Enter name here',
-]),
-acf_clone([
-	'label' => 'nameClone',
-	'name' => 'Name',
-	'clone' => [
-		0 => 'field_f1fab926', 
-		// This is the key of the field or group we want to clone. We can get the key by right clicking the field/group in the administrator dashboard and inspecting the element. In this basic example the only settings that will be copied is the placeholder.
-	],
+        'key' => 'field_nameInput',
```
Comment (vinkla) [inline README.md]: Please rename the key to `field_name` instead. We want be consistent with our naming convention.

**Comment by vinkla** [comment]
Thanks!

Happy Hacktoberfest!

**Pull request merged**